labs/cloud-pak-aiops/runbook-automation/7-try-it-out/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: 7. Try out your runbook
+description: Inject a synthetic alert and run your runbook
+sidebar_position: 7
+---
+
+## 3.1: Overview
+
+In this lab, you will inject a synthetic alert into your system, observe the runbook get associated with it, then run the runbook and observe parameters being passed, branching working, action output being passed from one action to another, and parameters being used in the target script.
+
+## 3.2 Set up the Netcool Connector integration
+
+If you haven't already completed the *Migration from Netcool* labs, you will need to follow the steps in module [3.2: Create a Netcool Connector instance](/waiops-tech-jam/labs/cloud-pak-aiops/migration-from-netcool/netcool-connector/#32-create-a-netcool-connector-instance) before you continue.
+
+Return to this point after you have set up the Netcool Connector instance.
+
+## 3.3: Inject a synthetic alert
+
+Use the following steps to inject a synthetic alert into AIOps:
+
+- Open a command terminal, connect to the virtual machine `netcoolvm` via SSH, and log in to the Netcool/OMNIbus ObjectServer:
+```
+ssh netcool@netcoolvm
+$OMNIHOME/bin/nco_sql -server AGG_P -user root -password netcool
+```
+
+- Copy and paste the following SQL into the SQL prompt:
+```
+insert into alerts.status (Identifier, Node, NodeAlias, Severity, Summary, Type, FirstOccurrence, 
+LastOccurrence, Class, Service, AlertGroup, Manager, ExpireTime) values 
+('BranchedExample', 'cisco1000', '', 5, 'Branched example: link down: cisco1000/slot1', 1, 
+getdate, getdate, 40057, 'Banking', 'CiscoLink', 'SNMP', 86400);
+go
+```
+- Type: `exit` and press enter to exit the command prompt
+- Log in to the AIOps console and select **Alerts** from the main menu
+- Type: `Branched` into the Search box at the top of the Alerts viewer
+
+You should see your synthetic alert in your Alerts viewer:
+
+![](images/synthetic-alert.png)
+
+:::note
+If you don't see the alert in your Alert viewer, wait up to a minute and then click the refresh button to refresh your view. The synthetic alert is being created in the Netcool/OMNIbus ObjectServer and then propagates up to AIOps via the Netcool Connector.
+:::
+
+- Under the **Runbooks** column, double-click the blue hyperlink: **1**
```

Comment:
No runbook associated:
![image](https://github.com/user-attachments/assets/1fa711b5-051e-45e3-8fe9-0a77f86ce8c1)

In earlier lab the policy was set to:
![image](https://github.com/user-attachments/assets/db7093fe-e7f6-47b1-9436-98e5fa576f19)

The alert summary in the synthetic event is: `Branched example: link down: cisco1000/slot1`

Changing policy to `Operator: starts with` associates the runbook correctly on insert:

![image](https://github.com/user-attachments/assets/e9aceefb-4362-4425-a5ed-d2974a16d174)
![image](https://github.com/user-attachments/assets/399e64e2-00eb-4160-b57d-c88f71077351)